spec/features/search/index_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'Search Controller' do
```

Comment:
This looks good. I would keep building on these tests and see about any sad paths. Like if you search something and it returns no results. You can try with your name.